File: src/test/models/EntityTest.java
package test.models;

import static org.junit.Assert.assertEquals;

import app.models.Entity;
import org.junit.Test;

public class EntityTest {

    @Test
    public void sampleModelTest() {
        Entity entity = new Entity("Bro");
        assertEquals("Bro", entity.getName());
    }
}

File: src/test/models/EntityTest.kt
package test.models

import app.models.Entity
import org.junit.Assert.assertEquals
import org.junit.Test

class EntityTest {
    @Test
    fun sampleTest() {
        val entity = Entity()
        entity.name = "Imported from java"
        assertEquals(entity.name, "Imported from java")
    }
}
